pkg/userdata/bottlerocket/node_userdata.go

```diff
@@ -14,12 +14,36 @@ import (
 )
 
 const (
-	adminContainerInitTemplate = `{{ define "adminContainerInitSettings" -}}
```

Comment:
What's happening to the admin container then? I see that we are introducing a field `AdminImage` above so how is that related to here.

Reply:
admin container is part of the hostContainers. We now loop through the host containers (admin, control, kubeadm-bootstrap) and generate the template